azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/middleware/sdk/ChatServiceListener.kt

```diff
@@ -74,13 +74,15 @@ internal class ChatServiceListener(
     private fun onMessagesPageModelReceived(
         messagesPageModel: MessagesPageModel,
         dispatch: Dispatch,
+        state: ReduxState
```

Comment:
would it be better for this function to take only threadId, not a whole state

Reply:
updated